apps/web/playwright/profile.e2e.ts

```diff
@@ -99,9 +99,7 @@ test.describe("Update Profile", () => {
     expect(await emailInputUpdated.inputValue()).toEqual(user.email);
   });
 
-  // TODO: This test is extremely flaky and has been failing a lot, blocking many PRs. Fix this.
-  // eslint-disable-next-line playwright/no-skipped-test
-  test.skip("Can update a users email (verification enabled)", async ({ page, users, prisma, features }) => {
```

Comment:
what did you do to fix this test?

Reply:
Cannot get this to fail locally.

Reply:
This it does fail, but very rarely. It is hard to track down. Will do it in a follow up pr.